### DISCUSSION TITLE
Any documentation or examples for API?

### DISCUSSION THREAD

**Discussion by Sovenok-Hacker** [ggml-org/llama.cpp#501] [Q&A] [answered]
I don`t understand how to use API (for example in Python with Ctypes).
I think some documentation would be helpful.

**Comment by Green-Sky** [comment, marked as answer]
i think you mean this? https://github.com/ggerganov/llama.cpp/issues/384

**Reply by Sovenok-Hacker** [reply]
Thanks.
